src/Analysis/Ast/Impl/Analyzer/PythonAnalyzer.cs

```diff
@@ -48,14 +49,15 @@ public sealed class PythonAnalyzer : IPythonAnalyzer, IDisposable {
         private PythonAnalyzerSession _currentSession;
         private PythonAnalyzerSession _nextSession;
 
-        public PythonAnalyzer(IServiceManager services) {
+        public PythonAnalyzer(IServiceManager services, string cacheFolderPath = null) {
```

Comment:
Should this argument be used at some `new PythonAnalyzer` site, i.e. with `DatabasePath` from the initialization options? My assumption is that we'd eventually have it removed from the extension in the future when the minimum version gets bumped up.

Reply:
Or, in testing, since I know we set other cache locations per test as well.

Reply:
Yes, setting was lost when I ported code from earlier work. Restored.